app/src/main/java/com/yml/womensafety/TipsForWomenSafetyAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package com.yml.womensafety
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.yml.womensafety.navigationdrawer.SafetyTips
+
+class TipsForWomenSafetyAdapter(private val tipsTextList: ArrayList<SafetyTips>) :
```

Comment:
Add a comment explaining about this class

Reply:
> Add a comment explaining about this class

Done

Reply:
I dont see the changes reflecting here. Are the changes pushed yet? @Deeksha1399

Reply:
I have pushed it to the other branch - Created Repository and ViewModel